integration-cli/docker_cli_run_test.go

```diff
@@ -3945,11 +3945,11 @@ func (s *DockerSuite) TestRunAttachFailedNoLeak(c *testing.T) {
 	assert.Assert(c, err != nil, "Command should have failed but succeeded with: %s\nContainer 'test' [%+v]: %s\nContainer 'fail' [%+v]: %s", out, err1, out1, err2, out2)
 	// check for windows error as well
 	// TODO Windows Post TP5. Fix the error message string
-	assert.Assert(c, strings.Contains(out, "port is already allocated") ||
-		strings.Contains(out, "were not connected because a duplicate name exists") ||
-		strings.Contains(out, "The specified port already exists") ||
-		strings.Contains(out, "HNS failed with error : Failed to create endpoint") ||
-		strings.Contains(out, "HNS failed with error : The object already exists"), fmt.Sprintf("Output: %s", out))
+	assert.Assert(c, strings.Contains(strings.ToLower(out), "port is already allocated") ||
```

Comment:
I would suggest to do
```go
out = strings.ToLower(out)
```
instead. Otherwise, this code is doing it five times :-\

Reply:
Also, test change would be better as a separate commit (preceding the one updating the vendor)